mk_verificator/tests/system/single_vip/test_single_vip.py

```diff
@@ -8,18 +8,21 @@
     utils.get_groups(utils.get_configuration(__file__))
 )
 def test_single_vip(local_salt_client, group):
+    config = utils.get_configuration(__file__)
+
     local_salt_client.cmd(group, 'saltutil.sync_all')
-    nodes_list = local_salt_client.cmd(group, 'grains.item', ['ipv4'])
+    nodes_list = local_salt_client.cmd(group, 'grains.item',
+                                       [config["protocol"]])
 
     ipv4_list = []
 
     for node in nodes_list:
-        ipv4_list.extend(nodes_list.get(node).get('ipv4'))
+        ipv4_list.extend(nodes_list.get(node).get(config["protocol"]))
```

Comment:
I suppose nope, at least if you wanna use not only ipv4 protocol then rename `ipv4_list` variable